README.md

```diff
@@ -1,2 +1,58 @@
 # Learning materials
-This repository contains learning materials, that is relevant for new engineers at Lunar.
+
+This repository contains learning materials, that are relevant for new engineers at Lunar.
+
+## Content
+
+1. Microservices
+2. Cloud Native
+3. Kubernetes and Containers
+4. Golang
+5. CI/CD and GitOps
+6. Observability
+7. Culture
+8. Other things
```

Comment:
Markdown lists are magical in that if you use `1.` for all items you don't need to update it when you add new items.

```suggestion
1. Microservices
1. Cloud Native
1. Kubernetes and Containers
1. Golang
1. CI/CD and GitOps
1. Observability
1. Culture
1. Other things
```